### DISCUSSION TITLE
Aligner network prior

### DISCUSSION THREAD

**Discussion by talipturkmen** [coqui-ai/TTS#1122] [General Q&A]
Hi,
In the aligner network implementation shouldn't we give prior by using the Hadamard product as stated in the paper instead of sum. 
https://github.com/coqui-ai/TTS/blob/c63bb481e95bd4a1ff978947d8e3e6c0bfb4177f/TTS/tts/layers/generic/aligner.py#L77

Thank you

**Comment by erogol** [comment]
Because they are in the log domain, it becomes a summation as far as I remember
